### ISSUE TITLE
Deprecation of text-davinci-003

### ISSUE THREAD

**Issue by Besarb** [MacPaw/OpenAI#144]
**Describe the bug**
When connecting to OpenAI API, the following error is shown ```APIErrorResponse(error: OpenAI.APIError(message: "The model `text-davinci-003` has been deprecated, learn more here: https://platform.openai.com/docs/deprecations", type: "invalid_request_error", param: nil, code: Optional("model_not_found")))```. 
"text-davinci-003" is indeed deprecated since January 4th 2024, see https://platform.openai.com/docs/deprecations.

**To Reproduce**
Try to connect to Open AI API using the latest version 0.2.5.

**Comment by iamyoungjo** [comment]
I think we need to have `gpt-3.5-turbo-instruct` supported.

**Comment by SunburstEnzo** [comment]
Yep can confirm, please pass in the raw value until support is added
https://github.com/MacPaw/OpenAI/issues/145#issuecomment-1925860639

**Comment by kalafus** [comment]
Options:
https://github.com/MacPaw/OpenAI/pull/166 updates the model, adds a deprecation warning.
https://github.com/MacPaw/OpenAI/pull/160 removes the model (due to frequent return of truncated responses; while the endpoint technically remains operational, i'm not sure what purposes it could still be useful for in its present state).

**Comment by nezhyborets** [comment]
Model removed in https://github.com/MacPaw/OpenAI/pull/160/
